llvm/docs/GettingStarted.rst

```diff
@@ -295,6 +295,7 @@ Package                                                     Version      Notes
 `python <http://www.python.org/>`_                          >=3.8        Automated test suite\ :sup:`1`
```

Comment:
Side note: This looks like it's different from what is posted at https://github.com/llvm/llvm-project/blob/888ae9066da9c9758c03e839be6cefb3246a9b1c/libc/docs/dev/header_generation.rst?plain=1#L25? Should we just adjust the version on the header generation page to say 3.8 given that is the LLVM project-wide minimum?

Reply:
SGTM

Reply:
Yes, I can edit the header generation page to match